src/inputs/Multi.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div class="form__radios">
```

Comment:
this is not a list of radio buttons; for styling purposes it should have it's own BEM class.